server/interfaces/interfaces.go

```diff
@@ -982,7 +982,7 @@ type Stdio struct {
 
 // CommandResult captures the output and details of an executed command.
 type CommandResult struct {
-	// Error is populated only if the command was unable to be started, or if it was
+	// InitError is populated only if the command was unable to be started, or if it was
```

Comment:
I think this name is more confusing than the current name, because this error could also be set if the command is SIGKILL'd or times out (both happen after initialization)

Maybe we could call it NonExitError or something?